graylog2-inputs/src/main/java/org/graylog2/inputs/transports/AmqpConsumer.java

```diff
@@ -120,9 +123,17 @@ public void handleDelivery(String consumerTag, Envelope envelope, AMQP.BasicProp
                         sourceInput.processRawMessage(rawMessage);
                         channel.basicAck(deliveryTag, false);
                     } catch (Exception e) {
-                        LOG.error("Error while trying to process AMQP message, requeuing message", e);
+                        LOG.error("Error while trying to process AMQP message", e);
                         if (channel.isOpen()) {
-                            channel.basicNack(deliveryTag, false, true);
+                            channel.basicNack(deliveryTag, false, requeueInvalid);
+
+                            if (LOG.isDebugEnabled()) {
```

Comment:
Small nitpick: The guard condition is unnecessary because the actual debug logging and string interpolation will only happen if debugging is enabled.